lob-java-examples/src/main/java/com/lob/examples/CsvUSVerificationExample.java

```diff
@@ -0,0 +1,59 @@
+package com.lob.examples;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+import com.lob.client.AsyncLobClient;
+import com.lob.client.LobClient;
+import com.lob.protocol.request.USVerificationRequest;
+import com.lob.protocol.response.USVerificationResponse;
+
+public class CsvUSVerificationExample extends BaseExample {
+    public static void main(final String[] args) throws Exception {
+        final LobClient client = AsyncLobClient.createDefault(API_KEY);
+
+        final CsvMapper mapper = new CsvMapper();
+
+        final CsvSchema schema = mapper.schemaFor(CsvAddress.class).withHeader();
+        final MappingIterator<CsvAddress> iterator = mapper.reader(CsvAddress.class).with(schema).readValues(fileFromResource("addresses/input.csv"));
+        while (iterator.hasNext()) {
+            final CsvAddress address = iterator.next();
+
+            final USVerificationRequest addressRequest = USVerificationRequest.builder()
+                    .primary_line(address.primary_line)
+                    .secondary_line(address.secondary_line)
+                    .city(address.city)
+                    .state(address.state)
+                    .zipCode(address.zip_code)
+                    .build();
+
+            final USVerificationResponse addressResponse = client.verifyUSAddress(addressRequest).get();
```

Comment:
Not anymore. The new US Verification always returns a HTTP 200 and and the response states whether the verification was successful or not.